edm4eic.yaml

```diff
@@ -372,39 +391,40 @@ datatypes:
   
   edm4eic::TrackParameters:
     Description: "ACTS Bound Track parameters"
-    Author: "W. Armstrong, S. Joosten"
+    Author: "W. Armstrong, S. Joosten, J. Osborn"
     Members:
-      - int32_t           type              // Type of track parameters (-1/seed, 0/head, ...)
-      - edm4hep::Vector2f loc               // 2D location on surface
-      - edm4eic::Cov2f    locError          // Covariance on loc
-      - float             theta             // Track polar angle [rad]
-      - float             phi               // Track azimuthal angle [rad]
-      - float             qOverP            // [e/GeV]
-      - edm4eic::Cov3f    momentumError     // Covariance on theta, phi and qOverP
-      - float             time              // Track time [ns]    
-      - float             timeError         // Error on the time
-      - float             charge            // Particle charge
+      - int32_t              type              // Type of track parameters (-1/seed, 0/head, ...)
+      - uint64_t             surface           // Surface for bound parameters (geometryID)
+      - edm4hep::Vector2f    loc               // 2D location on surface
+      - float                theta             // Track polar angle [rad]
+      - float                phi               // Track azimuthal angle [rad]
+      - float                qOverP            // [e/GeV]
```

Comment:
Add pdg hypothesis associated with q, and propagation?

Reply:
What do you mean by "propagation"?

Reply:
It's needed in the Acts::Propagator and Acts::Stepper, which starts from TrackParameters, e.g. through https://github.com/acts-project/acts/blob/1f6dd58d096f89452c1f5d8d5da65b6ddbb7e0bd/Core/include/Acts/EventData/TrackParameters.hpp#L14

Reply:
Oh you were just referring to the pdg hypothesis, I misunderstood your comment as saying ",and propagation" was referring to something else needed in the `TrackParameters`. You meant that the pid hypothesis is associated to charge and the propagation in Acts - understood now